com.trollworks.gcs/src/com/trollworks/gcs/weapon/WeaponDamage.java

```diff
@@ -424,8 +479,22 @@ private void extractWeaponDamageBonus(Feature feature, Set<WeaponDamageBonus> se
         }
     }
 
-    private static Dice addDice(Dice left, Dice right) {
-        return new Dice(left.getDieCount() + right.getDieCount(), Math.max(left.getDieSides(), right.getDieSides()), left.getModifier() + right.getModifier(), left.getMultiplier() + right.getMultiplier() - 1);
+    private Dice addDice(Dice left, Dice right) {
+        
+        if(mOwner.mOwner.getDataFile().getSheetSettings().getDamageProgression() == DamageProgression.PHOENIX_D3){
```

Comment:
Perhaps you should only do this for weapons that are actually using d6. In GURPS, I think that is currently all of them, however, GCS itself supports any arbitrary number of die sides in its input (and as a GM, I've occasionally made use of this). Only when the side type is left out does GCS default it to d6 (i.e. 2d resolves to 2d6, but 2d4 stays at 2d4).

Of course, that would mean that it would be impossible to actually specify d6's once this option was turned on, as you'd always downgrade them to d3's...

Reply:
Yeah, this is a good point. I've got an idea, I could make this code agnostic and have the 'convert to d3s' option becomes "convert dice down" or something; so the default behaviour is "turn disparate sided added dice into equivalents of the higher sided die" and then with the option on it's "turn added disparate dice into equivalents of the *lower* sided die". What do you think?

Reply:
Some sort of generic transformation would be good, if possible. Those titles sound awful, though. ;-)

There's also the standard GURPS option for turning adds into dice already present, which this might interact weirdly with.

As an aside, have you actually tried this new method in play? I think my players would rapidly revolt due to the annoyance of having to mentally convert all of those d6's on the table into d3 values. It isn't hard, but it can be tedious, especially if you have to do it every time you roll damage. I guess they do sell specialty dice that are labeled 1-3 twice, which would fix that... although I doubt my players would willingly buy them.

Reply:
Well, there's actually various d3s you can get; there's dedicated d3 shapes. There's also 6 sided die that only have 1 to 3; saves the mental conversion! We've only done some tests at this point, rather than like a full campaign with this though.